server/resources/resource_management.go

```diff
@@ -316,6 +316,23 @@ func GetMultipleResources(context middleware.Context, dataStore db.DB, resourceS
 	currCond := policy.GetCurrentResourceCondition()
 
 	filter := FilterFromQueryParameter(resourceSchema, queryParameters)
+	if search_filter,ok := queryParameters["search_field"]; ok && resourceSchema.IsSubstringSeachEnabled(){
+			for _,column := range search_filter{
+				if search_value,ok := queryParameters[column]; ok {
+					delete(filter, column)
+					searchValue := strings.Builder{}
+					searchValue.WriteString("~")
+					searchValue.WriteString(search_value[0])
+					filter[column] = searchValue.String()
+				} else{
+					err := fmt.Errorf("search value for `%s` not available in URL ", column)
+					return ResourceError{err, err.Error(), WrongQuery}
+				}
+			}
+			log.Info("substring search is enabled")
+	}
+
+	extendFilterByTenantAndDomain(resourceSchema, filter, schema.ActionRead, currCond, auth)
```

Comment:
`extendFilterByTenantAndDomain` is doubled.

Reply:
Yep, this is already called 2 lines below.